arbitrum-docs/how-arbitrum-works/03-sequencer.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Sequencer
+---
+
+## The Sequencer and Censorship Resistance
```

Comment:
Why not use this bit as the title? I'd say it's important to highlight the "censorship resistance" bit here.